tests/e2e/cypress/integration/Resources-status/resourceStatus.ts

```diff
@@ -0,0 +1,112 @@
+import { Given, And, When, Then } from 'cypress-cucumber-preprocessor/steps';
+
+import {
+  containerStateFilter,
+  selectFilterOnInput,
+  labelClearFilter,
+} from './resourceCommon';
+
+const serviceName = 'Ping';
+const resourcesRegexApiUrl = /\/api\/.+\/monitoring\/resources?/;
+
+Given('a valid centreon user account', () =>
+  cy.getCookie('PHPSESSID').should('exist'),
+);
+And('there is a page with a list of resources under monitoring', () =>
+  cy.get('table.MuiTable-root th p.MuiTypography-root').contains('Resource'),
+);
+And('the user can access this page', () =>
+  cy.get('nav[aria-label="Breadcrumb"]').should('be.visible'),
```

Comment:
It’s better to test that the message indicating that the user doesn’t have access to the page isn’t displayed